actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -432,6 +433,8 @@
   public static final String SUNBIRD_URL_SHORTNER_ENABLE = "sunbird_url_shortner_enable";
   public static final String USER_PROFILE_CONFIG = "userProfileConfig";
   public static final String PUBLIC_FIELDS = "publicFields";
+  public static final String PROFILE_USERTYPE = "profileUserType";
+  public static final String PROFILE_LOCATION = "profileLocation";
```

Comment:
It should be profileLocations, please check.

Reply:
but in the design it is only profileLocation, so should we change that